.github/workflows/problem-matchers/doxygen.json

```diff
@@ -10,7 +10,7 @@
       "owner": "doxygen",
       "pattern": [
         {
-          "regexp": "^\\/src\\/([^:]+):(\\d+): ?(\\w+): ?(.*)$",
+          "regexp": "^.*?\\/src\\/([^:]+):(\\d+): ?(\\w+): ?(.*)$",
```

Comment:
I deleted my original comment because I subsequently I thought I'd misread `.*?` as `.+?`, which is why I asked about grouping, since I believe the latter wouldn't have worked as expected, at least not reliably.

But then I realised you just changed the `+` to an `*`, presumably in response to my comment!